doc/bookkeeper-options.md

```diff
@@ -17,3 +17,28 @@ spec:
       codahaleStatsOutputFrequencySeconds: "30"
 ...
 ```
+### Bookkeeper JVM Options
+
+It is also possible to tune the Bookkeeper JVM by passing customized JVM options. Bookkeeper JVM Options
+are obvisouly for Bookkeeper JVM whereas aforementioned BookKeeper options are for BookKeeper server configuration.
+
+The format is as follows:
+```
+...
+spec:
+  bookkeeper:
+    bookkeeperJVMOptions:
+      memoryOpts: ["-Xms2g", "-XX:MaxDirectMemorySize=2g"]
+      gcOpts: ["-XX:MaxGCPauseMillis=20"]
+      gcLoggingOpts: ["-XX:NumberOfGCLogFiles=10"]
+      # extraOpts are options that don't belong to the previous categories. If you
+      # are not sure what categories that your option belongs to, please check if your option
+      # has a default in the Operator. If so, you must specify your option in the corresponding category
+      # in order to override the default one.
+      extraOpts: []
+...
+```
+The reason that we are using such detailed names like `memoryOpts` is because the Bookkeeper official [scripts](https://github.com/apache/bookkeeper/blob/master/bin/common.sh#L118) are using those and we need to override it using the same name.
+
+There are a bunch of default options in the Pravega operator code that is good for general deployment, please check [here](https://github.com/pravega/pravega-operator/blob/master/pkg/controller/pravega/bookie.go#L202). It is possible to override those default values by just passing the customized options. For example, the default option `"-XX:MaxDirectMemorySize=1g"` can be override by passing `"-XX:MaxDirectMemorySize=2g"` to
```

Comment:
We have not listed the default JVM Options set for bookkeeper by pravega-operator here. We need to do that.

Reply:
I made a link in doc so people can navigate to code and check the JVM options. I think it would be too much to list them all here, there is a lot defualt options